actionpack/lib/abstract_controller/rendering.rb

```diff
@@ -16,8 +16,22 @@ module Rendering
     extend ActiveSupport::Concern
     include ActionView::ViewPaths
 
-    # Normalizes arguments, options and then delegates render_to_body and
+    # Normalizes arguments, options and then delegates <tt>render_to_body</tt> and
     # sticks the result in <tt>self.response_body</tt>.
+    #
+    # The primary options are:
```

Comment:
Can you also add the options found here: http://guides.rubyonrails.org/layouts_and_rendering.html#using-render?

Mainly:
* `content_type`
* `layout`
* `status`
* `location`
* `formats`